src/app/controllers/test-controller.js

```diff
@@ -0,0 +1,9 @@
+angular.module('test', []).controller('TestController',
+  ['$scope', function($scope) {
+    $scope.countDown = function() {
+    if($scope.time) {
+      $scope.time -= 10;
+    }
+  };
+  $scope.time = 200;
```

Comment:
move line 8 to line 3 i.e. define `scope.time` before using it in `scope.countDown`.